Algebraic Topology/algebraic-topology.tex

```diff
@@ -224,18 +225,18 @@ \section{Homotopy}
 \end{thm}
 \begin{thm}
   If $X$ is path connected, then for any $x_a, x_b \in X$,
-  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$
-  notation If $X$ is path connected, we can denote the fundamental
+  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$ %
```

Comment:
So I am confused by this. My understanding was \Pi means fundamental groupoid and so we would say \Pi_1(X). Then, \pi_1(X,x) is the fundamental group at basepoint x. When it is clear that the fundamental group is the same regardless of basepoint, then we use \pi_1(X) as abusive shorthand. However, what is \Pi_1(X,x) ?